redbot/core/commands/context.py

```diff
@@ -28,7 +29,7 @@ def __init__(self, **attrs):
         self.permission_state: PermState = PermState.NORMAL
 
     async def send(self, content=None, **kwargs):
-        """Sends a message to the destination with the content given.
+        r"""Sends a message to the destination with the content given.
```

Comment:
`\*` down in the docstring below is an invalid escape character, and is technically deprecated since Python 3.6. Sometimes people escape `**kwargs` in docstrings with `\*\*kwargs` (same thing with `*args` -> `\*args`, thinking that sphinx will start bold/emphasis formatting. But sphinx actually handles this, and suggests omitting the backslashes for varargs: https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html

So, **TL;DR**, don't use `r"""` docstrings containing `\*\*kwargs`, use plain unicode `"""` docstrings containing `**kwargs`.

Reply:
My "why" was more, why not fix the root issue here, just not expressing it as such.